partials/tabs/mentors.html

```diff
@@ -1,59 +1,68 @@
 <div class="main-content container-fluid">
     <div class="row">
-        <div class="col-md-8 col-md-offset-2">
-            <div class="title-get-involved" >Hello there!</div>
-            <br>
-            <div class="description">
-                We are the mentors for coala in GSoC 2017.<br>
+        <div class="col m8 offset-m2">
+            <h1 class="fine center">Hello there!</h1>
+            <h4 class="light center">
+                We are the mentors for coala in GSoC 2017.
+            <h4>
+            <h4 class="light center"> 
                 Just drop a message on <a href="https://coala.io/chat" class="bear-link" >Gitter</a> -
-                we'd love to help you personally.<br>
-            </div>
+                we'd love to help you personally.
+            </h4>
+            <br>
         </div>
     </div>
 </div>
 
 <section>
     <div class="container">
-        <div class="row showcase-gi">
-            <div  ng-repeat="member in gic.mentorsList" class="card card-main showcase-card-gi">
-                <div class="empty"></div>
-                <div class="card-action profile-data" >
-                  <img class="center-block profile-img" height="100" ng-src="{{ member.github_avatar_url }}">
-                  <div class="person-name">{{ member.name }}</div>
-                  <div class="person-github-id-div"><a class="person-github-id" ng-href="https://github.com/{{member.github_handle}}">@{{ member.github_handle }}</a></div>
-                  <div class="person-github-bio">{{ member.bio }}</div>
+        <div class="row user">
+            <div class="parent-wrapper">
+                <div class="parent">
+                    <div ng-repeat="member in gic.mentorsList" class="card child card-main sc">
+                        <div class="empty"></div>
+                        <div class="data thumbnail center" >
+                          <img class="center dp" height="100" ng-src="{{ member.github_avatar_url }}">
+                          <div class="name center">{{ member.name }}</div>
+                          <div class="handle center"><a class="yellow-text text-darken-4" ng-href="https://github.com/{{member.github_handle}}">@{{ member.github_handle }}</a></div>
+                        </div>
+                    </div>
                 </div>
-            </div>
+            </div>        
```

Comment:
trailing whitespace in https://github.com/coala/projects/pull/297/commits/974c030aaa6423cf83dd866c289e61d2897455a4